hb-subset-plan.lst

```diff
@@ -0,0 +1,2 @@
+GRARNN: gr29643 used before defined gr29643 at:
+    0:      L4Z        gr29647=(hb_array_t<unsigned int>).length(gr29643,8)
```

Comment:
What's this change?

Reply:
seems this is somehow related to harfbuzz but it should be removed of course